### PR TITLE
Update download-avs script handle zip with a Carthage directory structure

### PR DESCRIPTION
**Pull request by typfel**

### CODE REVIEW

File: Wire-iOS.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -5322,12 +5322,12 @@
 				"$(SRCROOT)/Carthage/Build/iOS/ZMProtos.framework",
 				"$(SRCROOT)/Carthage/Build/iOS/ProtocolBuffers.framework",
 				"$(SRCROOT)/Carthage/Build/iOS/Cartography.framework",
-				"$(SRCROOT)/Carthage/Build/iOS/zmessaging.framework",
+				"$(BUILT_PRODUCTS_DIR)/zmessaging.framework",
```
Comment (daehn) [inline Wire-iOS.xcodeproj/project.pbxproj]: Can be removed?

File: Wire-iOS.xcodeproj/xcshareddata/xcschemes/Wire-iOS.xcscheme [hunk 2]
```diff
@@ -106,7 +106,7 @@
          </CommandLineArgument>
          <CommandLineArgument
             argument = "-ZMBackendEnvironmentType staging"
-            isEnabled = "NO">
+            isEnabled = "YES">
```
Comment (daehn) [inline Wire-iOS.xcodeproj/xcshareddata/xcschemes/Wire-iOS.xcscheme]: Can be removed?

File: Wire-iOS.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -5322,12 +5322,12 @@
...
 				"$(SRCROOT)/Carthage/Build/iOS/CocoaLumberjackSwift.framework",
 				"$(SRCROOT)/Carthage/Build/iOS/TTTAttributedLabel.framework",
 				"$(SRCROOT)/Carthage/Build/iOS/libPhoneNumber.framework",
 				"$(SRCROOT)/Carthage/Build/iOS/PINCache.framework",
-				"$(SRCROOT)/Carthage/Build/iOS/ZMCDataModel.framework",
+				"$(BUILT_PRODUCTS_DIR)/ZMCDataModel.framework",
```
Comment (daehn) [inline Wire-iOS.xcodeproj/project.pbxproj]: Can be removed?

File: Wire-iOS.xcodeproj/project.pbxproj (hunk 1 above)
Comment (typfel) [inline Wire-iOS.xcodeproj/project.pbxproj]: oops 😔 end of the day mistake

**Comment by typfel** [comment]
@daehn fixed your comments and made some more changes

**Pull request merged**